cmd/skopeo/sync.go

```diff
@@ -604,19 +610,22 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) error {
 			if err != nil {
 				return err
 			}
-
-			logrus.WithFields(logrus.Fields{
+			fromToFields := logrus.Fields{
 				"from": transports.ImageName(ref),
 				"to":   transports.ImageName(destRef),
-			}).Infof("Copying image ref %d/%d", counter+1, len(srcRepo.ImageRefs))
-
-			if err = retry.RetryIfNecessary(ctx, func() error {
-				_, err = copy.Image(ctx, policyContext, destRef, ref, &options)
-				return err
-			}, opts.retryOpts); err != nil {
-				return errors.Wrapf(err, "Error copying ref %q", transports.ImageName(ref))
 			}
-			imagesNumber++
+			if opts.dry {
+				logrus.WithFields(fromToFields).Info("Would have copied image ref %d/%d", counter+1, len(srcRepo.ImageRefs))
```

Comment:
```suggestion
				logrus.WithFields(fromToFields).Infof("Would have copied image ref %d/%d", counter+1, len(srcRepo.ImageRefs))
```
per the failing tests